R/player_transfer_history.R

```diff
@@ -12,7 +12,7 @@
 #'
 #' @export
 #'
-tm_player_transfer_history <- function(player_urls) {
+tm_player_transfer_history <- function(player_urls, get_extra_info = TRUE) {
```

Comment:
Awesome, much better defaulting this to `TRUE` so as not to cause any breaking changes by default.